src/AppInstallerRepositoryCore/Public/winget/RepositorySearch.h

```diff
@@ -134,6 +134,7 @@ namespace AppInstaller::Repository
         RelativePath,
         // Returned in hexadecimal format
         ManifestSHA256Hash,
+        Publisher,

```

Comment:
Looks like something I'll need to use in my change for matching heuristics. 🤔

Reply:
Yes, although it just makes getting the value easier since you don't have to get all of the metadata, so it isn't really critical to move to it.